src/rendering/batcher/gl/GlBatchAdaptor.ts

```diff
@@ -60,7 +61,14 @@ export class GlBatchAdaptor implements BatcherAdaptor
 
         renderer.shader.bind(this._shader, this._didUpload);
 
-        renderer.shader.bindUniformBlock(renderer.globalUniforms.uniformGroup, 'globalUniforms', 0);
+        if (renderer.type === RendererType.WEBGPU)
```

Comment:
why is the `gl` adapter checking for webgpu?

we have the `GpuBatchAdapter`

Reply:
good catch!

Reply:
fixed